site/src/pages/components/interest-invokers.explainer.mdx

```diff
@@ -4,145 +4,201 @@ name: Interest Invokers (Explainer)
-  on an element without invoking it, using a Human Input Device (HID). See below
-  for how various HIDs can show interest.
-- Loses Interest/Lost Interest: The action of _Loses Interest_ refers to the user
-  "moving away" from an element, or its _interestee_, using a HID; in other words
-  interest must be on a different element that is neither. Elements can only
-  _Lose Interest_ if they are in the state of Showing Interest. See below for
-  how various HIDs can lose interest.
-- Interestee: An element which is referenced to by an Interest element, via the
-  `interesttarget` attribute.
-
-
-### HIDs and interest
-
-Interest is shown and lost based on a user gesture, which will be dependent on
-the users Human Input Device (HID). A keyboard will use focus to determine
-interest equivalent to the `focusin`/`focusout` events available today. A
-pointer device, such as a mouse, will use the `pointerenter`/`pointerleave`.
-
-"Interest" is an intentional abstraction from traditional keyboard and pointer
-events to allow other HIDs to map an appropriate (privacy preserving) user
-gesture, without emulating or otherwise conflicting with keyboard or pointer
-events. For example a user agent with a touch screen HID _might_ map showing
-interest to a menu item in the devices long-press context-menu, while a device
-with hand tracking (such as mixed reality headsets) might offer an explicit hand
-gesture.
-
-While the proposal aims to encompass alternate HIDs and decvices, the specifics
-precisely how these HIDs will show Interest is not part of the initial proposal,
-and is an area of active discussion. It also stands to reason that
-to-be-invented devices may introduce new and novel concepts of showing/losing
-interest.
-
-## Proposed Plan
-
-In the style of `commandfor`, this document proposes we add `interesttarget`
-as available attributes to `<button>`, `<a>`, `<area>` and `<input>` elements.
+The [Invoker Commands API](../invokers.explainer), consisting of the `command` and `commandfor` attributes, provides an easy declarative way to make buttons directly trigger actions on a target element, such as showing a popover or a modal dialog when the button **is clicked**. This API, `interesttarget`, is very similar, in that it also provides a declarative way to invoke actions on a target element. However, rather than being activated via a **click** on the element, this API uses a lighter-touch way for the user to "show interest" in an element without fully activating it. For example, a link `<a href="..." interesttarget=foo>` element can be **hovered** with the mouse, which might trigger a hovercard preview of the link, without actually navigating to the link destination URL. Of course, if the link is **clicked**, then a normal navigation will occur.
+
+The `interesttarget` capability is supported on many interactive elements:
+- `<button>`
+- `<input type=button>`
+- `<input type=reset>`
+- `<input type=submit>`
+- `<a>`
+- SVG `<a>`
+- `<area>`.
```

Comment:
Perhaps worth calling out explicitly that it needs to be a `<a href="...">`  or the area equivalent? At least that's the current plan (though not in the original chrome prototype yet)?